src/components/SsoProviderSelector/SsoProviderSelector.tsx

```diff
@@ -0,0 +1,42 @@
+import type { FC } from "react";
+import EntryIdLogo from "./assets/EntryIdLogo";
+import KeycloakLogo from "./assets/KeycloakLogo";
+import OryHydraLogo from "./assets/OryHydraLogo";
+import Auth0Logo from "./assets/Auth0Logo";
+import SsoProviderButton from "./SsoProviderButton";
+
+const SsoProviderSelector: FC = () => {
+  return (
+    <div className="sso-provider-selector u-flex-column">
+      <p>Choose an SSO provider</p>
+      <div className="sso-provider-column u-flex-column">
+        <div className="sso-provider-row u-flex">
+          <SsoProviderButton
+            providerName="Entra ID"
+            to="https://documentation.ubuntu.com/lxd/latest/howto/oidc_entra_id/#"
```

Comment:
I think we should introduce the `useDocs` helper here. And then link to the local documentation if available with it.